src/coreclr/src/jit/instr.cpp

```diff
@@ -1053,9 +1052,11 @@ void CodeGen::inst_RV_TT_IV(instruction ins, emitAttr attr, regNumber reg1, GenT
             switch (rmOp->OperGet())
             {
                 case GT_LCL_FLD:
+                {
```

Comment:
I'm pretty sure I removed some such useless case braces, please do not add them back.

Reply:
Removed, as they were added by the IDE; but I find these braces helpful. It allows most editors to collapse an individual case which can help with reading/browsing source code.